lib/rucio/daemons/tracer/kronos.py

```diff
@@ -127,6 +138,17 @@ def __update_atime(self):
         rses = []
         for report in self.__reports:
             try:
+                # Identify suspicious files
+                if self.__bad_files_patterns and report['eventType'] in ('get_sm', 'sm_get', 'get_sm_a', 'sm_get_a') and report['clientState'].find('FAILED') > -1:
```

Comment:
Are you sure that get all needed clientStates with this? Looking at the last 30 days only at `get_sm/_a` I can see a lot of different error states and they do not all have "FAILED" in the name. Here is a list of all clientStates in the last 30 days that are not `DONE`, `FOUND_ROOT` or `ALREADY_DONE` ordered by count.

- `STAGEIN_ATTEMPT_FAILED`
- `NO_REPLICA`
- `STAGEIN_FILE_FAILED`
- `COPY_ERROR`
- `STAGEIN_FAILED`
- `NO_FILE`
- `CP_TIMEOUT`
- `GLOBUS_FAIL`
- `AD_MISMATCH`
- `BAD_COPYTOOL`
- `STAGEIN_BAD_COPYTOOL`
- `NO_SPACE`
- `NO_PROTOCOL_DEFINED`
- `ServiceUnavailable`
- `FILE_EXISTS`

Reply:
Where all the states are documented ?